### PR TITLE
update the consolidated customer profile api name

### PR DESCRIPTION
**Pull request by sramanan**
@tscanlin @nchilada - please review.

### CODE REVIEW

File: src/content/rest/customer_profiles/customer_profiles/3-consolidated.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 template: sidebyside
-endpoint: consolidated_customer_profile_view/567/oeu1234.5678
-endpoint_prefix: consolidated_customer_profile_view/
+endpoint: consolidatedCustomerView/567/oeu1234.5678
```
Comment (tscanlin) [inline src/content/rest/customer_profiles/customer_profiles/3-consolidated.md]: are all these endpoints getting camel-cased?

Comment (nchilada) [inline src/content/rest/customer_profiles/customer_profiles/3-consolidated.md]: @tscanlin I think this particular endpoint has always been camel-cased, and it's just the documentation that has been off.

But you bring up a good point: it's a little strange to expose /customer_profile and /consolidatedCustomerView (and /targetingEmbed).  @sramanan can we just rename the /consolidatedCustomerView endpoint to /consolidated_customer_view?

Comment (tscanlin) [inline src/content/rest/customer_profiles/customer_profiles/3-consolidated.md]: +1

Comment (sramanan) [inline src/content/rest/customer_profiles/customer_profiles/3-consolidated.md]: we could, That would require a vissandara deploy - i dont think there is anything out there already locked on to the name.

I would say lets push this doc change through and look at that as a separate issue.

Comment (nchilada) [inline src/content/rest/customer_profiles/customer_profiles/3-consolidated.md]: @sramanan okay.  We can easily revert this documentation change when we change the endpoint.

**Pull request merged**